.github/workflows/behat-tests.yml

```diff
@@ -0,0 +1,229 @@
+
+      - name: SSH agent
+        uses: webfactory/ssh-agent@v0.9.0
+        with:
+          ssh-private-key: ${{ secrets.SITE_OWNER_SSH_PRIVATE_KEY }}
+          log-public-key: true
+
+      - name: Configure SSH (no strict host key checking)
+        run: |
+          mkdir -p ~/.ssh
+          printf "Host *\n  StrictHostKeyChecking no\n  UserKnownHostsFile=/dev/null\n" > ~/.ssh/config
+
+      - name: Set Pantheon site id
+        run: echo "TERMINUS_SITE=wp-saml-auth" >> $GITHUB_ENV
+
+      - name: Compute Pantheon env vars
+        env:
+          SSP: ${{ matrix.ssp }}
+          TERMINUS_SITE: ${{ env.TERMINUS_SITE }}
+        run: |
+          set -e
+          ssp_num="${SSP//./}"
+          suffix=$(printf "%04d" $(( GITHUB_RUN_NUMBER % 10000 )) )
+          env_id="ci${ssp_num}${suffix}"
+          TERMINUS_ENV="${env_id:0:10}"
+          echo "TERMINUS_ENV=$TERMINUS_ENV" >> $GITHUB_ENV
+          echo "SIMPLESAMLPHP_VERSION=$SSP" >> $GITHUB_ENV
+          echo "PANTHEON_SITE_URL=${TERMINUS_ENV}-${TERMINUS_SITE}.pantheonsite.io" >> $GITHUB_ENV
+          echo "Using env: $TERMINUS_SITE.$TERMINUS_ENV -> https://${TERMINUS_ENV}-${TERMINUS_SITE}.pantheonsite.io"
+
+      - name: Install Composer deps
+        run: composer install --no-progress --no-interaction
+
+      - name: Prepare Behat environment
+        env:
+          WORDPRESS_ADMIN_USERNAME: pantheon
+          WORDPRESS_ADMIN_EMAIL: no-reply@getpantheon.com
+          WORDPRESS_ADMIN_PASSWORD: pantheon
+          WORKING_DIR: ${{ github.workspace }}
+        run: |
+          set -eux
+          if [ "${{ matrix.ssp }}" = "1.18.0" ]; then
+            bash bin/1.18/behat-prepare-simplesaml1.18.0.sh
+          else
+            bash bin/behat-prepare.sh
+          fi
+
+      - name: Wait for Pantheon workflow
+        run: |
+          SITE_ENV="${TERMINUS_SITE}.${TERMINUS_ENV}"
+          terminus workflow "$SITE_ENV"
+
+      - name: Wait until site responds
+        run: |
+          set -e
+          URL="https://${PANTHEON_SITE_URL}"
+          echo "Waiting for $URL ..."
+          for i in $(seq 1 60); do
+            if curl -fsSL -o /dev/null "$URL"; then
+              echo "OK"; exit 0
+            fi
+            sleep 5
+          done
+          echo "Site never became ready: $URL"
+          exit 1
+
+      - name: Run Behat
+        env:
+          WORDPRESS_ADMIN_USERNAME: pantheon
+          WORDPRESS_ADMIN_EMAIL: no-reply@getpantheon.com
+          WORDPRESS_ADMIN_PASSWORD: pantheon
+        run: |
+          set -e
+          echo ""
+          echo "=========================================================================="
+          echo "Running Behat on https://${PANTHEON_SITE_URL}/wp-login.php"
+          echo "with SimpleSAMLphp version ${SIMPLESAMLPHP_VERSION}"
+          echo "=========================================================================="
+          echo ""
+          ./bin/behat-test.sh --strict
```

Comment:
We are missing the `behat-cleanup.sh` step which deletes multidevs after tests pass. We need this back.